packages/tables/src/styled/StyledHeaderRow.ts

```diff
@@ -5,17 +5,42 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
-import classNames from 'classnames';
-import TableStyles from '@zendeskgarden/css-tables';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import stripUnit from 'polished/lib/helpers/stripUnit';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { StyledBaseRow, IStyledRowProps } from './StyledRow';
+import { StyledOverflowButton } from './StyledOverflowButton';
 
 const COMPONENT_ID = 'tables.header_row';
 
-export const StyledHeaderRow = styled.tr.attrs({
+const getHeaderRowHeight = (props: IStyledRowProps & ThemeProps<DefaultTheme>) => {
+  if (props.size === 'large') {
+    return `${props.theme.space.base * 18}px`;
+  } else if (props.size === 'small') {
+    return `${props.theme.space.base * 10}px`;
+  }
+
+  return `${props.theme.space.base * 12}px`;
+};
+
+export const StyledHeaderRow = styled(StyledBaseRow).attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(TableStyles['c-table__row'], TableStyles['c-table__row--header'])
+  'data-garden-version': PACKAGE_VERSION
 })`
+  border-bottom-color: ${props => getColor('neutralHue', 300, props.theme)};
+  height: ${getHeaderRowHeight};
+  vertical-align: bottom;
+  font-weight: ${props => props.theme.fontWeights.semibold};
+
+  ${StyledOverflowButton} {
+    opacity: 1;
+    margin-top: 0;
+    margin-bottom: calc(${props => stripUnit(getHeaderRowHeight(props)) / 2} - 1em);
```

Comment:
It looks like `calc` requires calculated values to have a unit, otherwise, the browser complains that the CSS value is invalid.

<img width="577" alt="Screen Shot 2020-01-02 at 10 54 54 AM" src="https://user-images.githubusercontent.com/1811365/71686088-12d77d80-2d4f-11ea-9c1d-60edd144d70d.png">

Reply:
Good catch! I've moved to the `math()` polished helper to retain the unit